test/testcafe/spec/ChallengeAuthenticatorEmail_spec.js

```diff
@@ -413,6 +424,19 @@ test
     await t.expect(resendEmailView.innerText).eql('Haven\'t received an email? Send again');
   });
 
+test
+  .requestHooks(logger, validOTPWithoutResendMock)('resend timer resets remediation has no resend context', async t => {
+    const challengeEmailPageObject = await setup(t);
+    await challengeEmailPageObject.clickEnterCodeLink();
+
+    await t.expect(challengeEmailPageObject.resendEmailView().hasClass('hide')).ok();
+
+    // The /poll simulates a response with a remediation with no "resend" context which will reset
+    // resend timer, hence the entry should be deleted from sessionStorage.
+    await t.wait(6000);
```

Comment:
Would be great to not add 6 real seconds to the test run time. Is there a way to mock this, or use configuration to decrease the poll time?

Reply:
I created a ticket to address this tech debt we have in testcafe..   https://oktainc.atlassian.net/browse/OKTA-430216

Reply:
Good point, I've updated the mocks/test to greatly reduce this now.